src/renderer/components/ProjectCreation/ModalContainer.tsx

```diff
@@ -92,6 +94,16 @@ const ModalContainer = ({ isOpen, closeModal }: Props) => {
             setProjectName={setProjectName}
           />
         );
+      case CloudConfigView:
+        return (
+          <CloudConfigView
+            prevView={prevView}
+            closeModal={showCancelProject}
+            nextView={nextView}
+            projectName={projectName}
+            textToDisplay={null}
```

Comment:
You can just make this an optional prop instead of explicitly passing null

Reply:
I tried 

```typescript
interface Props {
  prevView: () => void;
  closeModal: () => void;
  nextView: () => void;
  projectName: string;
  textToDisplay?: string | null;
}
```

But it errored with
```
propType "textToDisplay" is not required, but has no corresponding defaultProps declaration
```

[I didnt quite understand what the documentation for this error was trying to say](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/require-default-props.md)/I couldnt find a way around it

Reply:
The suggestion is to give it a default value, the page you linked described how. If you can't get your head around it don't worry, null is fine